opensrp-reveal/src/main/assets/json.form/case_confirmation.json

```diff
@@ -0,0 +1,316 @@
+        "expanded": false,
+        "min_date": "today-1y",
+        "max_date": "today",
+        "v_required": {
+          "value": "true",
+          "err": "Please enter the case notification date"
+        },
+        "relevance": {
+          "rules-engine": {
+            "ex-rules": {
+              "rules-file": "case-confirmation-relevance.yml"
+            }
+          }
+        }
+      },
+      {
+        "key": "caseInvestigationDate",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "date_picker",
+        "hint": "When was the case investigated? *",
+        "expanded": false,
+        "min_date": "today-1y",
+        "max_date": "today",
+        "v_required": {
+          "value": "true",
+          "err": "Please enter the case investigation date"
+        },
+        "relevance": {
+          "rules-engine": {
+            "ex-rules": {
+              "rules-file": "case-confirmation-relevance.yml"
+            }
+          }
+        }
+      },
+      {
+        "key": "forestGoerYesNo",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "native_radio",
+        "label": "Is this person a forest goer?",
+        "options": [
+          {
+            "key": "Yes",
+            "text": "Yes"
+          },
+          {
+            "key": "No",
+            "text": "No"
+          }
+        ],
+        "value": "No",
+        "relevance": {
+          "rules-engine": {
+            "ex-rules": {
+              "rules-file": "case-confirmation-relevance-rules.yml"
+            }
+          }
+        }
+      },
+      {
+        "key": "otherFocusYesNo",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "native_radio",
+        "label": "Is this case linked to any other focus?",
+        "options": [
+          {
+            "key": "Yes",
+            "text": "Yes"
+          },
+          {
+            "key": "No",
+            "text": "No"
+          }
+        ],
+        "value": "No",
+        "relevance": {
+          "rules-engine": {
+            "ex-rules": {
+              "rules-file": "case-confirmation-relevance-rules.yml"
+            }
+          }
+        }
+      },
+      {
+        "key": "otherFocusValue",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "openmrs_data_type": "text",
+        "type": "tree",
+        "hint": "Which focus?",
+        "tree": [],
+        "relevance": {
+          "rules-engine": {
+            "ex-rules": {
+              "rules-file": "case-confirmation-relevance-rules.yml"
+            }
+          }
+        }
+      },
+      {
+        "key": "comment",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "edit_text",
```

Comment:
This should be able to accept multiple sentences. Please review the "type"

Reply:
This type should be able handle multiple sentences, we shall provide a property to make it span several lines as required

Reply:
Please point me to something that already has a span and I can figure that out.